sigstore/_verify.py

```diff
@@ -136,6 +137,10 @@ def verify(
 
         `expected_cert_oidc_issuer` is the expected OIDC Issuer Extension within `certificate`.
 
+        `offline_rekor_entry` is an optional offline `RekorEntry` to verify against. If supplied,
```

Comment:
nit to add in comment: this only does SignedEntryTimestamp verification, but not inclusion proof verification. offline verification is slightly weaker and does not provide guarantees that the log has a consistent view

Reply:
94db410 should elaborate on this!